tools/cellxgene_census_builder/tests/test_manifest.py

```diff
@@ -37,6 +37,16 @@ def test_load_manifest_does_dedup(manifest_csv_with_duplicates: str) -> None:
         assert len(manifest) == 2
 
 
+def test_manifest_dataset_block(tmp_path: pathlib.Path, manifest_csv: str) -> None:
+    blocklist_fname = f"{tmp_path}/blocklist.txt"
+    blocklist_content = """# a comment\n\ndataset_id_1\n\n"""
+    with open(blocklist_fname, "w+") as f:
+        f.writelines(blocklist_content.strip())
+
+    manifest = load_manifest(manifest_csv, blocklist_fname)
+    assert len(manifest) == 1
```

Comment:
nit: I would also assert explicitly that `dataset_id_1` is not admitted, and maybe generate the blocklist by taking a known dataset id from the manifest itself (in case the manifest csv test fixture is changed in the future).